src/Sarif/Core/PropertyBagHolder.cs

```diff
@@ -63,6 +63,8 @@ public string GetProperty(string propertyName)
                         propertyName));
             }
 
+            if (Properties[propertyName] == null) { return null; }
```

Comment:
>Properties [](start = 16, length = 10)

This fixes a crash when you attempted to deserialize a null-valued property (`"aString": null`) into a string with `GetProperty("aString")`.